internal/cmd/scraper/scraper.go

```diff
@@ -59,7 +59,6 @@ func validateConfig(cfg *Config) error {
 
 // LoadViperDefaults loads the default configuration into the given Viper loader.
 func LoadViperDefaults(viper *viper.Viper) {
```

Comment:
WDYT about moving this to the same file as the `loadConfig` function?

Reply:
good one! done